portfolio/src/main/webapp/style.css

```diff
@@ -4,6 +4,91 @@
   width: 650px;
 }
 
-#greeting-container {
-  margin-top: 20px;
+#pf-image {
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+  width: 50%;
+}
+
+h1, h2 {
+  display: flex;
+  justify-content: center;
+  text-transform: uppercase;
+}
+
+.icon-row {
+  overflow: auto;
+  text-align: center;
+  width: 100%;
+}
+
+.image-container {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+}
+
+.single-container, .image-container img {
+  max-height: 200px;
+  max-width: 200px;
+}
+
+.single-container {
+  margin: 10px;
+  position: relative;
+}
+
+.slide-container:hover .img-block, .single-container:hover .img-block {
+  opacity: .70;
 }
+
+.img-block {
+  background-color: blue;
+  bottom: 0;
```

Comment:
nit: This can be removed because the max-height is overriding the bottom position